src/main/java/com/mitchej123/hodgepodge/mixins/Mixins.java

```diff
@@ -953,6 +953,11 @@ public enum Mixins implements IMixins {
     FIX_HEALING_AXE_HEAL(new MixinBuilder("Fix the healing axe not healing entities when attacking them")
             .addMixinClasses("extrautilities.MixinItemHealingAxe").setPhase(Phase.LATE).setSide(Side.BOTH)
             .setApplyIf(() -> FixesConfig.fixExtraUtilitiesHealingAxeHeal).addTargetedMod(TargetedMod.EXTRA_UTILITIES)),
+    FIX_CHEST_COMPARATOR_UPDATE(
+            new MixinBuilder("Fix comparator redstone signal updates for mini and slightly larger chests")
```

Comment:
This should be changed to match the config comment:
```suggestion
            new MixinBuilder("Fix Extra Utilities chests not updating comparator redstone signals when their inventories change")
```